types/index.test-d.ts

```diff
@@ -1,5 +1,5 @@
 import { expectType, expectError } from 'tsd'
-import sjson = require('.')
+import sjson from '..'
```

Comment:
Why `import sjson = require('.')` to `import sjson from '.'`?

Reply:
I got a TS 1202 Error

![image](https://user-images.githubusercontent.com/5059100/181746018-7a9a76d7-9ea1-4df2-a4a6-ffb1cb3bc07b.png)

So I thought: Why not.